packages/bot-web-ui/package.json

```diff
@@ -23,7 +23,9 @@
         "build": "f () { npm run build:skeleton && webpack --progress --env base=$1 ;}; f",
         "build:skeleton": "lerna exec --scope @deriv/bot-skeleton -- npm run build",
         "build:travis": "echo \"No build:travis specified\"",
-        "test:eslint": "eslint \"./src/**/*.?(js|jsx|ts|tsx)\"",
+        "test:eslint": "eslint \"./src/**/*.?(js|jsx|ts|tsx|spec.ts|spec.js|spec.tsx|spec.tsx)\"",
```

Comment:
I checked that if we do not specify test's(rtl/jest)extensions here, those files will not fall under the check and fix the imports, they will just be ignored. That is why I wrote them all here manually. But the interesting fact is that in the Eslint configuration we do not have to specify the same extensions(test's(rtl/jest)extensions).

Reply:
can we use this regex pattern `/(?<!spec)\.(ts|tsx|js|jsx)$/gm`